lib/grandpa/grandpa.go

```diff
@@ -507,6 +507,8 @@ func (s *Service) playGrandpaRound() error {
 
 	logger.Debugf("sending pre-vote message %s...", pv)
 	roundComplete := make(chan struct{})
+	// <-roundComplete will receive the default value of channel's type when it gets
+	// closed, so we don't need to explicitly send a value.
```

Comment:
```suggestion
	// roundComplete is a signal channel which is closed when the round completes
```

It's commonly called signal channels :wink:

Reply:
rephrased the comment with this suggestion